documents.go

```diff
@@ -43,13 +64,13 @@ func (rc *RockClient) AddDocuments(ctx context.Context, workspace, collection st
 	})
 
 	if err != nil {
-		return nil, err
+		return openapi.AddDocumentsResponse{}, err
 	}
 
 	log := zerolog.Ctx(ctx)
 	logDocumentStatuses(log.Trace(), resp.GetData()).Msg("added documents")
 
-	return resp.GetData(), nil
+	return *resp, nil
```

Comment:
I'd generally expect these to be almost interchangeable.  Or to return the data + offset separately (i.e. 2 return values) but you would know more how go users would use this.

Reply:
they added a new field to the response struct, so in case another field is added in the future this will cover that case (instead of having to add a third method)